coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -5598,7 +5755,16 @@ def std(context, node):
 @register_torch_op
 def copy(context, node):
     inputs = _get_inputs(context, node, expected=[2, 3])
-    context.add(mb.identity(x=inputs[0], name=node.name))
+    if context.frontend == TorchFrontend.TORCHSCRIPT:
+        result = mb.identity(x=inputs[0], name=node.name)
+    elif context.frontend == TorchFrontend.EXIR:
+        src = inputs[1]
```

Comment:
A comment here about why EXIR is different than Torch Script, would be good.

Reply:
Unfortunately I also don't know why in torch script `torch.copy` is a no op 😢

In EXIR, `torch.copy(x, y)` results in `y` (may get broadcast to `x` shape), which makes perfect sense

Reply:
hmmm ... 
Yeah prob we added that graph pass after the initial implementation of `def copy_` ...
If your assumption is correct, that we never hit the `TorchFrontend.TORCHSCRIPT` in this function,
we would suggest add an assertion here and remove the legency implementation (`result = mb.identity(x=inputs[0], name=node.name)`).
Also adding a comment about why this function is never hit with torchscript frontend would be good ..

Reply:
Yeah probably 😂 I found `def copy` is introduces 3 years ago, then `def _internal_op_tensor_inplace_copy` is 2 years ago

Lemme try remove the legacy implementation and kick off a CI

Reply:
[CI](https://gitlab.com/coremltools1/coremltools/-/commit/2849faf6d445304df4bd9eed38758bb08a742c55/pipelines) ✅ Indeed we were never using `def copy` in torch script 😆